website/api/controllers/webhooks/receive-from-github.js

```diff
@@ -25,8 +25,23 @@ module.exports = {
     let GitHub = require('machinepack-github');
 
     let GREEN_LABEL_COLOR = 'C2E0C6';// « Used in multiple places below.
-    let GITHUB_USERNAMES_OF_BOTS = [// « Used in multiple places below.
-      'sailsbot'
+    let GITHUB_USERNAMES_OF_BOTS = [// « Used in multiple places below.  (actually, some maintainers too, not just bots)
```

Comment:
Is this used anywhere outside of this file? If not, maybe we can change the name to something like `SKIP_NOTIFICATIONS_GITHUB_USERNAMES`?

Reply:
yeah you're right we should, was just being lazy

Reply:
thanks!